client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -392,7 +408,16 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 	const isPluginBundleEligible = useIsPluginBundleEligible();
 	const isBundled = selectedDesign?.software_sets && selectedDesign.software_sets.length > 0;
 
+	const selectedDesignTier =
+		useSelector(
+			( state ) =>
+				selectedDesignThemeId && getThemeTierForTheme( state, selectedDesignThemeId )?.slug
+		) ?? FREE_THEME;
```

Comment:
[Nitpick, non-blocking] I still think this is hard to mentally parse. I personally find easier to understand this with more lines:

```suggestion
	const selectedDesignTier = useSelector( ( state ) => {
		if ( ! selectedDesignThemeId ) {
			return FREE_THEME;
		}
		
		const themeTier = getThemeTierForTheme( state, selectedDesignThemeId );
		return themeTier?.slug ?? FREE_THEME;
	} );
```

Reply:
I prefer the current implementation, thanks for providing a diff!